app/Dao/Patient/PatientDao.php

```diff
@@ -2,12 +2,33 @@
 
 namespace App\Dao\Patient;
 
-use App\Contracts\Dao\Patient\PatientDaoInterface;
+use App\Models\Patient;
 use Illuminate\Http\Request;
+use App\Contracts\Dao\Patient\PatientDaoInterface;
 
 /**
  * Data accessing object for patient
  */
 class PatientDao implements PatientDaoInterface
 {
+    /**
+     * To show create patient view
+     * 
+     * @return View patients
+     */
+    public function index()
+    {
+        $patients =  Patient::all();
+        return $patients;
+    }
+
+    /**
+     * To show patient by id
+     * @return View patients
+     */
+    public function show($id)
+    {
+        $patient = Patient::find($id);
```

Comment:
If param-id not match in Patient id, show 404 page.
sample code :
`$patient = Patient::findOrFail($id);`
OR,
`$patient = Patient::where('id', $id)->firstOrFail();`